### PR TITLE
LOG-1700: Fix ES-Proxy master to version 1.0

### PR DESCRIPTION
**Pull request by vimalk78**
### Description
  Parent Task:  [Make operand images reusable across openshift-logging releases](https://issues.redhat.com/browse/LOG-1635)
 This PR fixes the `master` branch of ES_Proxy to version "v1.0". Once this is merged, CPaaS Dockerfile wil be changes to reflext the same version.

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-1635
- Enhancement proposal:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,15 @@ vendor:
 .PHONY: vendor
 
 image:
-	podman build -f Dockerfile -t $(LOCAL_IMAGE_TAG) .
+	CI_CONTAINER_VERSION="v1.0" \
```
Comment (jcantrill) [inline Makefile:50]: Where do we call this target other then as a developer?  CI has it's own config which calls `make build`.  That does not seem to trace to this target.  What will CPaaS do?  Does this need to be moved to maybe the Dockerfile.in?

Comment (vimalk78) [reply Makefile:50]: CI does not use `make build`
CI uses specified Dockerfile path  https://github.com/openshift/release/blob/master/ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-release-5.0.yaml#L63-L70

perhaps this change should go into `hack/generate-dockerfile-from-midstream` , we should be able to set all variables except commit_id. does CI has a way to capture it?

Comment (vimalk78) [reply Makefile]: moved the change to Dockerfile generation

Comment (jcantrill) [reply Makefile]: > CI does not use `make build`

CI uses `make build` via the Dockerfile https://github.com/openshift/elasticsearch-proxy/blob/master/Dockerfile.in#L19 which will call the default target https://github.com/openshift/elasticsearch-proxy/blob/master/Makefile#L30

File: Makefile [hunk 2]
```diff
@@ -2,6 +2,11 @@ export GOROOT=$(shell go env GOROOT)
 export GOFLAGS=-mod=vendor
 export GO111MODULE=on
 
+ES_PROXY_MAJOR="1"
```
Comment (jcantrill) [inline Makefile:5]: Is there any reason not to add these directly to the Dockerfile.in?  There is nothing about the changes where we are calculating version

Comment (vimalk78) [reply Makefile]: updated

**Review by jcantrill — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**